app/controllers/api/api_controller.rb

```diff
@@ -49,5 +50,9 @@ def bad_request_response(exception)
     def invalid_transition(exception)
       render json: { errors: Api::ParamErrorFactory.new(error: I18n.t(:invalid_transition), params: exception).call }, status: :unprocessable_entity
     end
+
+    def invalid_updated_since_response(exception)
+      render json: { errors: Api::ParamErrorFactory.new(error: I18n.t(:invalid_updated_since_filter), params: exception).call }, status: :bad_request
```

Comment:
Maybe the title could remain `bad_request`, and the error message could be `invalid_updated_since_filter` instead, to be more consistent with the above errors?